features/zones/Create.feature

```diff
@@ -89,6 +91,19 @@ Feature: zones / create
     And the "$egress-input-switch input" element doesn't exist
     And the "$environment-universal-config" element contains "globalAddress: grpcs://<global-kds-address>:5685"
 
+  Scenario: The zone get connected successfully and shows the success
+    Given the environment
+      """
+      KUMA_SUBSCRIPTION_COUNT: 0
+      """
+    And the URL "/provision-zone" responds with
+      """
+      body:
+        token: spat_595QOxTSreRmrtdh8ValuoeUAzXMfBmRwYU3V35NQvwgLAWIU
+      """
+    When I visit the "/zones/-create" URL
+    And I "type" "test" into the "$name-input" element
+    And I click the "$create-zone-button" element
```

Comment:
Previously the below assertion was merged in with a scenario titled `The form interactions behave correctly`. I wanted it to be clearer that we are testing whether we show that the zone connects successfully so I split the test into two. So the green above here is just the same setup copy/pasta'ed